learn/experimental/reduce-indexing-memory-usage.mdx

```diff
@@ -0,0 +1,13 @@
+# Reduce indexing memory usage
+
+The `--experimental-reduce-indexing-memory-usage` configuration option enables `MDB_WRITEMAP`, an LLMDB option. Activating it can help reducing RAM usage in some UNIX and UNIX-like setups. However, it may also negatively impact write speeds and overall performance.
```

Comment:
```suggestion
The `--experimental-reduce-indexing-memory-usage` configuration option enables `MDB_WRITEMAP`, an LMDB option. Activating it can help reduce RAM usage in some UNIX and UNIX-like setups. However, it may also negatively impact write speeds and overall performance.
```